MediaAdminBundle/Resources/public/ecmascript/OpenOrchestra/Application/Collection/Media/Medias.js

```diff
@@ -40,9 +40,16 @@ class Medias extends DataTableCollection
      */
     _getSyncUrl(method, options) {
         let urlParameter = options.urlParameter || {};
+        if ('read' == method && true == urlParameter.withoutPerimeter) {
+            method = 'read-without-perimeter';
+        }
+        delete urlParameter.withoutPerimeter;
+
         switch (method) {
             case "read":
-                return Routing.generate('open_orchestra_api_media_list', urlParameter);
+                return Routing.generate('open_orchestra_api_media_list_with_perimeter', urlParameter);
+            case "read-without-perimeter":
```

Comment:
i think is better to use the same mechanism than other model . and use the api context